scripts/windows_install_openssl.ps1

```diff
@@ -13,7 +13,7 @@ $ErrorActionPreference = "Stop"
 $filepath = "$Env:TEMP/openssl-setup.exe"
 
 echo "downloading openssl"
-curl.exe -o "$filepath" -sSL https://slproweb.com/download/Win64OpenSSL-1_1_1q.exe
+curl.exe -o "$filepath" -sSL https://slproweb.com/download/Win64OpenSSL-1_1_1s.exe
```

Comment:
Can you open a separate PR for this? We need to merge it soon.

Reply:
PR Link: https://github.com/anarazel/pg-vm-images/pull/33